admin/test/test_release.py

```diff
@@ -1130,6 +1134,9 @@ def test_real_dpkg_utils(self):
         """
         Calling :func:`update_repo` with real dpkg utilities creates a
         repository in S3.
+
+        The filenames in the respository metadata does not have the build directory
```

Comment:
'respository' -> 'repository'
'does not have' -> 'do not have'
Also this line is > 80 characters